include/SDL.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <SDL2/SDL.h>
+
+class SDL {
+public:
+    SDL();
+    bool init_SDL();
+    void close();
+    SDL_Window *get_Window() const { return window; };
+    SDL_Renderer* get_Renderer() const { return renderer; };
```

Comment:
`get_window()` and `get_renderer()` might not have to be const. main.cpp should ultimately have full control over the window and renderer. 

Also, 
`SDL_Renderer* get_Renderer() const { return renderer}; `
should be
`SDL_Renderer *get_Renderer() const { return renderer}; ` 
:)